src/Thorn_Calculus-Core_Bisimilarities.thy

```diff
@@ -1541,8 +1541,7 @@ proof (coinduction rule: synchronous.up_to_rule [where \<F> = "[\<sim>\<^sub>s]
       "R = post_receive n Y (\<lambda>y. A \<triangleright>\<^sup>\<infinity> x. \<P> x \<parallel> \<Q> y) \<parallel> (B \<triangleright>\<^sup>\<infinity> y. (A \<triangleright>\<^sup>\<infinity> x. \<P> x \<parallel> \<Q> y)) \<guillemotleft> suffix n"
       by (auto elim: transition_from_repeated_receive)
     have "B \<triangleright>\<^sup>\<infinity> y. \<Q> y \<rightarrow>\<^sub>s\<lparr>B \<triangleright> \<star>\<^bsup>n\<^esup> Y\<rparr> post_receive n Y \<Q> \<parallel> (B \<triangleright>\<^sup>\<infinity> y. \<Q> y) \<guillemotleft> suffix n"
-      using receiving [where \<P> = "\<lambda>y. \<Q> y \<parallel> B \<triangleright>\<^sup>\<infinity> y. \<Q> y", unfolded post_receive_after_parallel]
-      by (subst repeated_receive_proper_def) force
+      by (fact repeated_receive_transition)
```

Comment:
This can be simplified to `using repeated_receive_transition .`.

Reply:
Fixed in 334f4bb7454f9e0671ab3fd3fb31b8e7c37b0a52, please verify.

Reply:
Looks good.